net/api/impl/posix_server.h

```diff
@@ -190,6 +192,30 @@ class HTTPServerPOSIX final {
     handlers_.erase(path);
   }
 
+  struct StaticFileServer {
+    std::string body;
+    std::string content_type;
+    explicit StaticFileServer(const std::string& body, const std::string& content_type)
+        : body(body), content_type(content_type) {}
+    void operator()(Request r) { r.connection.SendHTTPResponse(body, HTTPResponseCode.OK, content_type); }
```

Comment:
Only `GET` requests must be served, other should response with `Method Not Allowed`